mod-linked-data/src/main/resources/citation/mod-linked-data/features/authority/authority-update.feature

```diff
@@ -0,0 +1,67 @@
+Feature: Authority update
+
+  Background:
+    * url baseUrl
+    * callonce login testUser
+    * def testUserHeaders = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': '*/*' }
+    * configure headers = testUserHeaders
+    * def authorityFileRequest = read('samples/authority_source_file.json')
+    * call postAuthorityFile
+
+  Scenario: create authority and work, then update authority
+    # Step 1: create an authority
+    * def authorityRequest = read('samples/authority_person.json')
+    * def postAuthorityCall = call postAuthority
+    And match postAuthorityCall.response.qmRecordId == '#notnull'
+
+    # Step 2: search for the created authority
+    * def query = 'headingRef < "PAVELTEST" or headingRef >= "PAVELTEST"'
+    * def searchAuthorityCall = call searchAuthority
+    And match searchAuthorityCall.response.items[0].authority.id == '#notnull'
+    * def authorityId = searchAuthorityCall.response.items[0].authority.id
+
+    # Step 3: get SRS id of found authority
+    * def idType = 'AUTHORITY'
+    * def getAuthoritySrsCall = call getSourceRecordFormatted
+    And match getAuthoritySrsCall.response.id == '#notnull'
+    * def authoritySrsId = getAuthoritySrsCall.response.id
+
+    # Step 4: create a work linking it to the authority by SrsId
+    * def resourceRequest = read('samples/work_with_authority.json')
+    * def postWorkCall = call postResource
+    * def workResponse = postWorkCall.response.resource['http://bibfra.me/vocab/lite/Work']
+    And match workResponse.id == '#notnull'
+    And match workResponse._creatorReference[0].label == 'PAVELTEST'
+    * def workId = workResponse.id
+
+    # Step 5: update the authority adding a birth date
+    * def authorityUpdateRequest = read('samples/authority_person_update.json')
+    * def putAuthorityCall = call putAuthority
+
+    # Sleep for 5 seconds to let kafka messages be handled
+    * sleep(5)
```

Comment:
is it possible to replace this ```sleep()``` call with ```retry until```?

Reply:
Well, I think it's better to make a test wait for 5 seconds instead of spamming requests to the test environment, because there are many other teams running their Karate tests